src/components/vm/hostdevs/hostDevCard.jsx

```diff
@@ -95,40 +95,40 @@ function getSource(hostDev, nodeDevices, hostdevId) {
 
         // If there are 2 usb devices without specified bus/device and same vendor/product,
         // it's impossible to identify which one is the one referred in VM's XML
-        cells.push(getOptionalValue(device, `${hostdevId}-device`, _("Device")));
-        cells.push(getOptionalValue(bus, `${hostdevId}-bus`, _("Bus")));
+        cells.push(getOptionalValue(device, `device-${hostdevId}`, _("Device")));
+        cells.push(getOptionalValue(bus, `bus-${hostdevId}`, _("Bus")));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1162-20230808-102821-f5d4e2e6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)